spec/connect/cli_spec.rb

```diff
@@ -293,9 +293,11 @@
       subject.new(argv)
     end
 
-    it 'output help on help flag' do
+    it 'outputs help on help flag with no line longer than 80 characters' do
       argv = %w{--help}
-      expect_any_instance_of(subject).to receive(:puts)
+      expect_any_instance_of(subject).to receive(:puts) do |option_parser|
+        expect(option_parser.instance_variable_get(:@opts).to_s.split("\n").map(&:length).max).to be <= 80
```

Comment:
I find `option_parser.instance_variable_get(:@opts)` very, very ugly... I don't even know why it works.

What do you think of this solution? http://stackoverflow.com/a/16507680/686769 (check the *RSpec < 3.0 (and others)* section, method `capture_stdout`).